hooks/tk-shell_actions.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+Hook that loads defines all the available actions, broken down by publish type.
+"""
+import pprint
+import sgtk
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class ShellActions(HookBaseClass):
```

Comment:
Good point. I've renamed the actions to be named Debug Action 1 and so on. I'd rather not have them be hidden behind a debug descriptor however. I think it is implied that these apps make no sense outside a DCC and as such the hooks are there for testing. I'll ask @robblau about it.